tests/eosiod_run_test.py

```diff
@@ -323,25 +329,48 @@ def cmdError(name, code=0, exitNow=False):
     node.waitForTransIdOnNode(transId)
 
     Print("Get transaction details %s" % (transId))
-    transaction=node.getTransaction(transId)
+    transaction=None
+    if not enableMongo:
+        transaction=node.getTransaction(transId)
+    else:
+        if amINoon:
+            transaction=node.getActionFromDb(transId)
+        else:
+            transaction=node.getMessageFromDb(transId)
     if transaction is None:
         cmdError("eosc get transaction trans_id")
         errorExit("Failed to retrieve transaction details %s" % (transId))
 
-    typeVal=  transaction["transaction"]["actions"][1]["name"]
-    amountVal=transaction["transaction"]["actions"][1]["data"]["amount"]
+    typeVal=None
+    amountVal=None
+    if amINoon:
+        if not enableMongo:
+            typeVal=  transaction["transaction"]["actions"][1]["name"]
+            amountVal=transaction["transaction"]["actions"][1]["data"]["amount"]
+        else:
+            typeVal=  transaction["name"]
+            amountVal=transaction["data"]["amount"]
+    else:
+        if not enableMongo:
+            typeVal=  transaction["transaction"]["messages"][0]["type"]
+            amountVal=transaction["transaction"]["messages"][0]["data"]["amount"]
+        else:
+            typeVal=  transaction["type"]
+            amountVal=transaction["data"]["amount"]
+
     if typeVal!= "transfer" or amountVal != 975311:
-    #if transaction.tType != "transfer" or transaction.amount != 975311:
         errorExit("FAILURE - get transaction trans_id failed: %s" % (transId), raw=True)
 
     Print("Get transactions for account %s" % (testeraAccount.name))
     actualTransactions=node.getTransactionsArrByAccount(testeraAccount.name)
     if actualTransactions is None:
         cmdError("eosc get transactions testera")
```

Comment:
Please search and change all `eosc` to `eosioc`, `eosd` to `eosiod`.  And `eos-walletd` to  `eosio-walletd` or `eosiowd` which will be the new name I believe.

Reply:
Valid comment on executable name change. Made the change.